liana-gui/src/installer/mod.rs

```diff
@@ -242,13 +241,31 @@ impl Installer {
 
     pub fn update(&mut self, message: Message) -> Task<Message> {
         match message {
-            Message::HardwareWallets(msg) => match self.hws.update(msg) {
-                Ok(cmd) => cmd.map(Message::HardwareWallets),
-                Err(e) => {
-                    error!("{}", e);
-                    Task::none()
+            Message::HardwareWallets(msg) => {
+                let update = matches!(&msg, &HardwareWalletMessage::List(_));
+                match self.hws.update(msg) {
+                    Ok(cmd) => {
+                        let task_1 = cmd.map(Message::HardwareWallets);
+                        let mut task_2 = Task::none();
+                        if update {
+                            task_2 = self
+                                .steps
+                                .get_mut(self.current)
+                                .expect("There is always a step")
+                                .update(
+                                    &mut self.hws,
+                                    // We notify downstream that the the list have been updated
+                                    Message::HardwareWallets(HardwareWalletMessage::Update),
```

Comment:
it is a nit:
I think HardwareWalletMessage should be reserved for internal use of the hws module.
I think a new Message::HardwareWalletsUpdated should be added to the installer Message instead as it is trigger by the installer.update() for its internal steps.